api/v1alpha1/datadogagent_types.go

```diff
@@ -358,6 +375,39 @@ type ProcessSpec struct {
 	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
 }
 
+// OrchestratorExplorerConfig contains the orchestrator explorer configuration.
+// The orchestratorExplorer runs in the process-agent and DCA.
+// +k8s:openapi-gen=true
+type OrchestratorExplorerConfig struct {
+	// Enable this to activate live Kubernetes monitoring.
+	// ref: https://docs.datadoghq.com/infrastructure/livecontainers/#kubernetes-resources
+	//
+	// +optional
+	Enabled *bool `json:"enabled,omitempty"`
+
+	// +optional
+	// Option to disable scrubbing of sensitive container data (passwords, tokens, etc. ).
+	Scrubbing *Scrubbing `json:"scrubbing,omitempty"`
+
+	// +optional
+	// Additional endpoints for shipping the collected data as json in the form of {"https://process.agent.datadoghq.com": ["apikey1", ...], ...}'.
+	AdditionalEndpoints *string `json:"additionalEndpoints,omitempty"`
+
+	// +optional
+	// Set this for the Datadog endpoint for the orchestrator explorer
+	DDUrl *string `json:"ddUrl,omitempty"`
+
+	// +optional
+	// Additional tags for the collected data in the form of `a b c`
+	ExtraTags *string `json:"extraTags,omitempty"`
```

Comment:
i think you are referring to `DD_TAGS`, we added this option for cluster agent wide settings. 
More information can be found here (@xlucas did the PR) in the pr in the comment: https://github.com/DataDog/datadog-agent/pull/6515#issuecomment-717252460

Reply:
thanks for the context:
* could you add more explanation in the comment? like the different with DD_TAGS 
* could you change the type for a []string. Even if after it is used as a string. From a user perspective, I feel it more natural to have a list. 

```suggestion
	ExtraTags []string `json:"extraTags,omitempty"`
```

Reply:
Added it 👍 , with this I needed to make some adjustments to the error handling because I needed to parse the list to a string like with DDTags.
https://github.com/DataDog/datadog-operator/blob/f5fdf1196f6b50d1266d4f7101ad8a635df5d834/controllers/datadogagent/utils.go#L567-L577